src/types/contract.rs

```diff
@@ -1,42 +1,160 @@
+pub use build_info::BuildInfo;
 use serde::{Deserialize, Serialize};
 
 /// The struct provides information about deployed contract's source code and supported standards.
 ///
-/// Contract source metadata follows [NEP-330 standard](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) for smart contract verification
+/// Contract source metadata follows [**NEP-330 standard**](https://github.com/near/NEPs/blob/master/neps/nep-0330.md) for smart contracts
+///
+/// See documentation of [`crate::Contract::contract_source_metadata`] on how to query this for a contract via this crate
+// `rustdoc` clearly lacks functionality of automatic backlinks within a single crate
 #[derive(Debug, Clone, PartialEq, Default, Eq, Serialize, Deserialize)]
 pub struct ContractSourceMetadata {
-    /// Optional version identifier, typically a commit hash or semantic version
+    /// Optional version identifier, typically a semantic version
+    ///
+    /// ## Examples:
     ///
-    /// Examples:
-    /// - Git commit: "39f2d2646f2f60e18ab53337501370dc02a5661c"
```

Comment:
Why to remove that? I believe, it's part of spec. 
Do you think it's not really make sense?

Reply:
in point of fact, 
i needed to somewhat supress what

```lang
#[near(contract_metadata(
    version = "random identifier",
    link = "https://github.com/org/repo",
    standard(standard = "xyz", version = "1.0.0"),
))]
```

snippet introduces for `link` attribute with value, generated 
by cargo-near,

but looking at it for the second time, there's no need at all to do this suppression for
`version`,
and for `link` it can be made flexible enough to sometimes use
value from `#[near(contract_metadata(...)` , though with lowest priority.

I'll add this example back and a comment on `link` too